test/unit-tests/Main.hs

```diff
@@ -0,0 +1,19 @@
+import Test.Tasty (defaultMain, testGroup)
+
+-- import qualified Fingerprint
+-- import qualified Fingerprint.Array
+import qualified Spectacle.Syntax.NonDet
+-- import qualified Spectacle.Syntax.Logic
+import qualified Spectacle.Syntax.Modal
+
+main :: IO ()
+main =
+  defaultMain $
+    testGroup
+      "unit tests"
+      [ -- Fingerprint.tests
+        -- , Fingerprint.Array.tests
```

Comment:
Is this commented out code coming back soon?

Reply:
Eventually yes, the tests for the fingerprint were slow so I pushed them out of the way for now. They'll come back with the model checker changes.